nise/util/__init__.py

```diff
@@ -43,15 +41,22 @@ def load_yaml(objekt):
 
 
 def deepupdate(original, update):
-    """Recursively update a dict.
+    """Recursively update a nested dict/list.
 
-    Subdict's won't be overwritten but also updated.
+    original and update must have the same structure
     """
-    if not isinstance(original, abc.Mapping):
+    if not (isinstance(update, list) or isinstance(update, dict)):
```

Comment:
This is a little difficult to read. Could it be changed for clarity?
```python
if not isinstance(update, (list, dict)):
    ...
```

Reply:
I'm open to suggestions on how to do that.

Would adding more comments to document the logic help?

Reply:
I've left a suggestion in the review comment. I just find that syntax easier to read and it's only one call to `isinstance`

Reply:
```suggestion
    if not isinstance(update, (list, dict)):
```